From 8170703201722b25dfae631fc6a6997e9f01a655 Mon Sep 17 00:00:00 2001
From: xantares <xantares09@hotmail.com>
Date: Tue, 19 Sep 2023 19:49:07 +0200
Subject: [PATCH] link metis to gklib

---
 libmetis/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libmetis/CMakeLists.txt b/libmetis/CMakeLists.txt
index fc6cec6..350d659 100644
--- a/libmetis/CMakeLists.txt
+++ b/libmetis/CMakeLists.txt
@@ -6,6 +6,7 @@ file(GLOB metis_sources *.c)

 # Build libmetis.
 add_library(metis ${METIS_LIBRARY_TYPE} ${metis_sources})
+target_link_libraries(metis GKlib)

 if(METIS_INSTALL)
   install(TARGETS metis
